app/views/dashboard/show.html.erb

```diff
@@ -1,11 +1,27 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-l">User dashboard</h1>
-    <p class="govuk-body"> You are logged in</p>
-
     <% if @current_user.lead_provider? %>
-      <%= govuk_link_to "Go to school search", school_search_path %>
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+          <%= @lead_provider.name %>
+        </h1>
+        <div class="index-group">
+          <h2 class="govuk-heading-m">
+<!--            TODO Fix after Ethan code merge for lead providers-->
+            <%= govuk_link_to "Find and add schools", nil %>
+          </h2>
+          <p class="govuk-body">Search for schools you want to recruit and add schools you have partnered with.</p>
+        </div>
+        <div class="index-group">
+          <h2 class="govuk-heading-m">
+            <%= govuk_link_to "Check your schools for 2021", lead_providers_your_schools_path(selected_cohort_id: @lead_provider.cohorts.find_by(start_year: Time.zone.now.year).id) %>
+          </h2>
```

Comment:
This seems a bit weird to me - hardcoding the year in the text but not in the link?

Reply:
Good spot will change

Reply:
Done